README.md

```diff
@@ -10,6 +10,9 @@ comments.  For more information about OpenShift, visit https://openshift.redhat.
 or the OpenShift support page
 https://openshift.redhat.com/support.
 
+RHC is typically built on Ruby 1.8.7.  If you are running into any dependency 
```

Comment:
This is true of the product, but anyone who is reading the README is probably getting RHC from Git in order to build it themselves.  Perhaps a better wording would be something like, "RHC can be built on Ruby 1.8.x, 1.9.x, and 2.x."